### DISCUSSION TITLE
Update of openssl

### DISCUSSION THREAD

**Discussion by jasontempleman-eaton** [git-for-windows/git#4106] [General]
This is using openssl 1.0.2 which was last updated 12/2019.  Are there any plane to update this to use the latest version?

**Comment by dscho** [comment]
I have no idea what Git for Windows version you use, but the latest version [ships](https://github.com/git-for-windows/build-extra/blob/main/versions/package-versions-2.38.1.txt#L49) [with](https://github.com/git-for-windows/build-extra/blob/main/versions/package-versions-2.38.1.txt#L100) [OpenSSL v1.1.1q](https://github.com/git-for-windows/build-extra/blob/main/versions/package-versions-2.38.1.txt#L119).

**Reply by rimrul** [reply]
> I have no idea what Git for Windows version you use

Looks like it's some version prior to 2.20.0
